ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCli.scala

```diff
@@ -284,7 +284,7 @@ class CommonCli(name: LedgerName) {
       .opt[String]("contract-id-seeding")
       .optional()
       .text(
-        s"""Set the seeding mode of contract IDs. Possible values are $allSeedingModeNames. Default is "$defaultSeedingModeName".""")
+        s"""Set the seeding mode of contract IDs. Possible values are $allSeedingModeNames. "$defaultSeedingModeName" mode is not compatible with development mode. Default is "$defaultSeedingModeName".""")
```

Comment:
Development mode is currently turned on by default in Sandbox and can't be turned off.

I think you may want to make it off-by-default and move the `--dev-mode-unsafe` switch from DAML-on-SQL's _Cli.scala_ to _CommonCli.scala_ so that Sandbox users can turn it on (if necessary).

Reply:
change. dev mode is now off by default and can be turned on manually.